smarty/data/atomtypes/basetypes.smarts

```diff
@@ -1,4 +1,6 @@
 % atom types
+[$ewg1] ewg1
+[*]     any_atom
```

Comment:
There may be other grouped atoms we should be including here. 
I think there are actually two electron withdrawing groups ($ewg1 and $ewg2) in parm@frosst. 
I'll take a look to see what makes the most sense for this part. 

Shouldn't these be listed as most general to least general? so the `[*]` option would come first.